.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
 - "./travis-tool.sh install_r stringr"
 - "./travis-tool.sh install_r testthat"
 - "./travis-tool.sh install_github jimhester/covr robertzk/testthatsomemore avantcredit/AWS.tools"
-- "./travis-tool.sh install_github kirillseva/cacher"
+- "./travis-tool.sh install_github kirillseva/cacher jimhester/withr"
```

Comment:
I think this would be covered by `./travis-tool.sh install_deps`. We could also get a more modern .travis.yml.